src/Microsoft.Health.Fhir.Api/Features/Operations/Import/InitialImportLockMiddleware.cs

```diff
@@ -38,7 +39,12 @@ public sealed class InitialImportLockMiddleware
             {
                 (HttpMethods.Get, ".*"), // Exclude all read operations
                 (HttpMethods.Post, ".*/\\$import"),
-                (HttpMethods.Delete, ".*/_operations/import/.+"),
+                (HttpMethods.Delete, ".*/_operations/.+/.+"), // Allow the cancelation of any long running job
+            };
+
+            _filteredEndpoints = new HashSet<(string method, string pathRegex)>()
+            {
+                (HttpMethods.Get, ".*/\\$reindex"), // New long running jobs shouldn't be started while import is running
```

Comment:
Not sure what's the export behavior if the resource already imported but indexesParams not yet imported.

Reply:
The export just uses the regular search functionality, so it would be the same as if you searched for the resource.